.buildkite/scripts/unit_test.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 
 set -euo pipefail
+set -o xtrace
```

Comment:
shall we avoid this? I've been thinking to somehow use some env variable to enable debugging a bit easier, but probably that's something to discuss in a follow-up

Reply:
Sorry, build is failing so I had to put these back again.  Will remove once CI is green on this PR.

Reply:
Thanks, removed in 13a31239.